cms/djangoapps/api/docs/decisions/001-course-apps-api.rst

```diff
@@ -0,0 +1,153 @@
+Course Apps API
+_______________
+
+Status
+======
+Proposal
+
+Context
+=======
+
+The new `Course Authoring MFE`_ includes a new UX called "Pages and Resources"
+for configuring course "apps" such as progress, wiki, teams, discussions etc.
+all from one place. Currently these apps are intermingled with the core
+platform and are inseparable from it, however future apps could be implemented
+as plugins.
+
+In order to drive this new UI, we need a way to discover installed and available
+course apps and provide an API that can be used by MFEs to display information
+related to these apps and potentially enable/disable these apps.
+
+.. _Course Authoring MFE: https://github.com/edx/frontend-app-course-authoring/
+
+
+Decision
+========
+
+We can develop a new course apps API that can be used to manage the apps
+available for a course. The list of *available* apps might not match the list
+of *installed* apps since it is possible that certain apps may be disabled, or
+otherwise unavailable for a particular course.
+
+The course apps API will only list the available apps, and allow toggling them
```

Comment:
Linking the [previous comment](https://github.com/edx/edx-platform/pull/27193#discussion_r607189465) here, as GitHub has marked it as outdated.

Reply:
The course apps API will ~~only~~ list the available apps and allow _users to_ toggling them on or off.